lib/sdf-server/src/service/v2/func/binding/create_binding.rs

```diff
@@ -288,7 +364,18 @@ pub async fn create_binding(
                             .await?;
                             let schema_variant =
                                 SchemaVariant::get_by_id_or_error(&ctx, schema_variant_id).await?;
-
+                            let func = Func::get_by_id_or_error(&ctx, func_id).await?;
+                            ctx.write_audit_log(
+                                AuditLogKind::AttachManagementFunc {
```

Comment:
Unrelated to your PR but this route is frightening with all the nested SHEESH! We should refactor this later...

Reply:
Yeah, definitely worth a pass.